addon/instance-initializers/clear-double-boot.js

```diff
@@ -29,7 +29,7 @@ export default {
 
       instance.didCreateRootView = function() {
         clearHtml();
-        originalDidCreateRootView.apply(instance, arguments);
+        originalDidCreateRootView.apply(this, arguments);
```

Comment:
@bobisjan Do you still have this issue?  Is this due to babel transpilation?  i.e. what happens if we convert the function to arrow syntax so it operates in the context of the enclosing scope?